package.json

```diff
@@ -21,6 +21,8 @@
     "dist:mac": "cross-env GENERATE_SOURCEMAP=false npm run build && npm run package:mac",
     "dist:win": "cross-env GENERATE_SOURCEMAP=false npm run build && npm run package:win",
     "test": "node scripts/test.js --env=node",
+    "coverage": "yarn test --coverage",
+    "report-coverage": "yarn coverage && cat ./coverage/lcov.info && codecov",
```

Comment:
not sure if this `cat` is necessary. I'd expect codecov to find the file itself

see https://github.com/codecov/codecov-node/blob/master/lib/codecov.js#L28

Reply:
Ah OK, I think you're right. I've seen a similar code [here](https://github.com/facebook/jest/issues/6563). But there it was piped from `cat` to `codecov`.
And here I think it's called separately after `cat` is executed.

I'll remove it.